### PR TITLE
Fix for bsc#1056045

### PR DESCRIPTION
**Pull request by fsundermeyer**
Added an openSUSE-specific procedure for creating a root image

### CODE REVIEW

File: lxc.xml [hunk 1]
```diff
@@ -0,0 +1,590 @@
...
+    <para>
+     Install a minimal system:
+    </para>
+    <screen>&prompt.root;zypper --root <replaceable>/PATH/TO/ROOTFS</replaceable> in -t pattern minimal_base</screen>
+   </step>
+   <step>
+    <para>
+     Set the &rootuser; password:
+    </para>
+    <screen>&prompt.root;echo "ttyS0" >>/PATH/TO/ROOTFS/etc/securetty
```
Comment (heijligen) [inline lxc.xml]: <screen>&prompt.root;echo "ttyS0" >><replaceable>/PATH/TO/ROOTFS</replaceable>/etc/securetty

File: lxc.xml [hunk 2]
```diff
@@ -0,0 +1,590 @@
...
+&prompt.root;echo "root:YOURPASSWD" | chpasswd -R /PATH/TO/ROOTFS</screen>
```
Comment (heijligen) [inline lxc.xml]: &prompt.root;echo "root:YOURPASSWD" | chpasswd -R \<replaceable>/PATH/TO/ROOTFS\</replaceable>\</screen>
<replaceable> tag is missing

**Comment by cbosdo** [comment]
The procedure and doc looks good to me, but I find suspicious that the commit doesn't remove anything, just adds code.

**Comment by heijligen** [comment]
The file is on a wrong position. (/lxc.xml not xml/lxc.xml) Thats the reson for not removing anything

**Comment by fsundermeyer** [comment]
Sorry, my bad - I moved the file to the correct location. 

It does not remove stuff, because I used DocBooks "conditional text" feature. The original stuff will only show up in builds for SLE (os="sles;sled") whereas the new stuff (os="osuse") will only show in builds for openSUSE.

**Comment by heijligen** [comment]
From which branch is the documentation at doc.opensuse.org created? there it should appear. I've marked you two lines in the document where you've forgotten to ad \<replaceable> tags

**Comment by cbosdo** [comment]
@fsundermeyer the commit looks way better now ;)

File: lxc.xml (hunk 1 above)
Comment (fsundermeyer) [reply lxc.xml]: Fixed with 4fef835

File: lxc.xml (hunk 2 above)
Comment (fsundermeyer) [reply lxc.xml]: Fixed with 4fef835

**Comment by fsundermeyer** [comment]
The missing replaceables have been added with 4fef835

Once this pull request is approved and merged, I will cherry-pick it into maintenance/openSUSE42.3 and update doc.o.o from there. This PR was branched off of the develop branch, which always needs to have all changes to make sure they do not get lost for the next openSUSE/SLE release.

**Comment by heijligen** [comment]
line 142 and 143: now there is one "replaceable" too much `TO/ROOTFS</replaceable>replaceable>/`

**Comment by fsundermeyer** [comment]
Sigh, I whish I wouldn't have to do ten things in parallel today.
Thanks for your patience.

**Comment by heijligen** [comment]
Thanks for the work! From my view it looks fine now

**Pull request merged**